Library/Formula/run.rb

```diff
@@ -0,0 +1,21 @@
+class Run < Formula
+  homepage "http://runscripts.org"
+  url "https://github.com/runscripts/run/archive/0.3.6.tar.gz"
+  sha1 "0f181b6a3bb56769cdbc46e6d0431a00d2cf3b9c"
+  head "https://github.com/runscripts/run.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    system "go", "get", "github.com/runscripts/run"
```

Comment:
Please look at the use of `go get` in other formulae and see how they symlink to avoid a new download. Thanks!